### PR TITLE
insomnia: Update to version 2021.1.1

### PR DESCRIPTION
**Pull request by lucasiro**
version 2021.1.1

### CODE REVIEW

**Comment by Ash258** [comment]
/verify Checkver has to fail

Insomnia and insomnia designer were merged together. Freeze insonmnia-designer manifest with notes about this change

**Comment by lucasiro** [comment]
insonmnia-designer manifest freezed with note an suggest

File: bucket/insomnia-designer.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "description": "Collaborative API Design Tool for designing and managing OpenAPI specs",
     "homepage": "https://insomnia.rest",
     "license": "MIT",
+    "notes": "Insomnia Designer and Insomnia Core were merged together. Please install insomnia",
+    "suggest": {
+        "Insomnia": [
+            "extras/insomnia"
+        ]
+    },
```
Comment (Ash258) [inline bucket/insomnia-designer.json:11]:
```suggestion
    "notes": "Insomnia Designer and Insomnia Core were merged together. Install 'insomnia' instead.",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
- Fix checkver for insomnia.json
    - Run `checkver insomnia -f` after
- Remove checkver and autoupdate from designer

**Pull request merged**